src/main/java/net/alpenblock/bungeeperms/Lang.java

```diff
@@ -287,6 +287,7 @@ public static String translatePreserveArgs(MessageType type, Object... vars)
         HELP_GROUP_REMOVETIMEDPERM("help.group.remove-timed-perm", "Removes a timed permission from the group"),
         HELP_GROUP_HAS("help.group.has", "Checks if the group has the permission"),
         HELP_GROUP_LIST("help.group.list", "Lists the permissions of the group"),
+        HELP_SEARCH("search.search", "Lists where a permission is set for groups and users"),
```

Comment:
Please also add the String to the [default language file](https://github.com/weaondara/BungeePerms/blob/master/src/main/resources/lang/en-GB.yml) (and any other languages you might know)